streams/src/main/java/org/apache/kafka/streams/processor/internals/StateDirectory.java

```diff
@@ -54,14 +61,27 @@
     private static final Logger log = LoggerFactory.getLogger(StateDirectory.class);
     static final String LOCK_FILE_NAME = ".lock";
 
+    /* The process file is used to persist the process id across restarts.
+     * The version 0 schema consists only of the version number and UUID
+     *
+     * If you need to store additional metadata of the process you can bump the version numberand append new fields.
+     * For compatibility reasons you should only ever add fields, and only by appending them to the end
+     */
+    private static final String PROCESS_FILE_NAME = "kafka-streams-process-metadata";
+    private static final int PROCESS_FILE_VERSION = 0;
```

Comment:
No idea if we'll ever want to add anything else to this file, but better to be safe and forward compatible than sad